components/config/histogram_column_picker.py

```diff
@@ -0,0 +1,22 @@
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+def get_component(columns):
+    """ Return a column picker component.
+
+    Keyword arguments:
+    columns -- The columns available in the uploaded data
+    """
+    return html.Div(children=[
+        html.Div(children='Column'),
+        dcc.Dropdown(
+            id='column',
+            clearable=False,
+            searchable=True,
+            options=[
+                {'label': column, 'value': column}
+                for column in columns
+            ],
+        )
+    ])
```

Comment:
It might be a good idea to use the existing code here, as this is almost duplicate to "axis_picker". Instead of a boolean there, you could just pass an ID string for the element, and then use that ID as a label too (possibly using some string manipulation, i.e. replacing all dashes with spaces, capitalizing). The axis_picker also already includes an option to pass a default value, which would be a good idea here, too (possibly magnitude?).